Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
+        return self.field(field)
+    }
+}
+
+
+struct Artist {
+    let id: Int?
+    // Note the optionality. Fields that are marked `NOT NULL` in your schema should be non-optional.
+    var name: String
+    var genre: String?
+
+    // You probably want your own initializer
+    init(name: String, genre: String) {
+        self.id = nil
+        self.name = name
+        self.genre = genre
+    }
+}
+
+
+extension Artist: Table {
```

Comment:
Could you explain the reasoning behind `Model` -> `Table`? Personally I think that making your model a `Model` makes more sense, but if you have a good reason I'm sure you can change my mind.